source/otus-user/src/main/java/br/org/otus/persistence/pendency/dto/UserActivityPendencyFieldOrderingOptions.java

```diff
@@ -0,0 +1,32 @@
+package br.org.otus.persistence.pendency.dto;
+
+public enum UserActivityPendencyFieldOrderingOptions {
+
+  STATUS("status"),
+  ACRONYM("acronym"),
+  RECRUITMENT_NUMBER("rn"),
+  EXTERNAL_ID("externalID"),
+  DUE_DATE("dueDate"),
+  REQUESTER("requester"),
+  RECEIVER("receiver");
+
+  private String name;
+
+  UserActivityPendencyFieldOrderingOptions(String name){
+    this.name = name;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public static boolean contains(String orderingOption){
+    for (UserActivityPendencyFieldOrderingOptions option : UserActivityPendencyFieldOrderingOptions.values()) {
+      if (option.getName().equals(orderingOption)) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
@FlaviaAP podemos deixar este cara mais elegante.

```
return UserActivityPendencyFieldOrderingOptions.values().stream()
    .map(UserActivityPendencyFieldOrderingOptions::getName)
    .anyMatch(orderingOption::equals);
```
Referencia: https://stackoverflow.com/questions/23004921/how-to-check-if-element-exists-using-a-lambda-expression